.github/workflows/docs.yml

```diff
@@ -38,7 +45,62 @@ jobs:
   build-and-deploy:
     uses: ./.github/workflows/build-and-deploy-docs.yml
     with:
+      git-branch: gh-pages
       version: ${{ needs.setup.outputs.version }}
       ref: ${{ needs.setup.outputs.ref }}
     needs:
       - setup
+
+  update-version-in-json-index:
+    runs-on: ubuntu-latest
+    needs:
+      - setup
+      - build-and-deploy
+
+    steps:
+      - name: Download source
+        uses: actions/checkout@v5
+        with:
+          ref: gh-pages
+
+      - name: Update version list
+        run: |
+          VERSION="${{ needs.setup.outputs.version }}"
+          IS_RELEASED="${{ needs.setup.outputs.is-released }}"
+
+          JQ_EXPRESSION="
+            .versions |= (
+              if ( . | map(.name) | index(\"${VERSION}\") | not ) then
```

Comment:
Will this remove or replace a version if it already exists?

Reply:
Conceptually it could be translated to Crystal as:

```cr
versions = 
  if !versions.map(&.name).includes?(VERSION)
    versions << Version.new(name: VERSION, url: "/ameba/#{VERSION}/", released: IS_RELEASED)
  else
    versions
  end
```